doc/source/rllib/rllib-rlmodule.rst

```diff
@@ -138,11 +138,11 @@ RLlib implements the following abstract framework specific base classes:
 
 The minimum requirement is for sub-classes of :py:class:`~ray.rllib.core.rl_module.rl_module.RLModule` is to implement the following methods:
 
-- :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule._forward_train`: Forward pass for training.
+- `_forward_train`: Forward pass for training.
```

Comment:
Why are we not linking to the method? I'm assuming that was the original intent.

Reply:
Because this is a private method, and `sphinx.ext.autosummary` does not document private methods by default - so any reference to this fill fail. If you want this to be a link, we need to explicitly tell `sphinx.ext.autosummary` to include this private method in the documentation, which we can totally do - it's up to you.

But if we do want this to be user facing, maybe it shouldn't be a private method...?

Reply:
Why would we change the behavior of `sphinx.ext.autosummary` for this one case? Maybe I'm missing something, but this doesn't seem like a clean solution.